Emby.Drawing/ImageProcessor.cs

```diff
@@ -48,18 +54,23 @@ public sealed class ImageProcessor : IImageProcessor, IDisposable
         /// <param name="fileSystem">The filesystem.</param>
         /// <param name="imageEncoder">The image encoder.</param>
         /// <param name="mediaEncoder">The media encoder.</param>
+        /// <param name="config">The configuration.</param>
         public ImageProcessor(
             ILogger<ImageProcessor> logger,
             IServerApplicationPaths appPaths,
             IFileSystem fileSystem,
             IImageEncoder imageEncoder,
-            IMediaEncoder mediaEncoder)
+            IMediaEncoder mediaEncoder,
+            IConfiguration config)
         {
             _logger = logger;
             _fileSystem = fileSystem;
             _imageEncoder = imageEncoder;
             _mediaEncoder = mediaEncoder;
             _appPaths = appPaths;
+            _config = config;
+
+            _parallelEncodingLimit = new(_config.GetParallelImageEncodingLimit());
```

Comment:
Isn't this the wrong constructor? Should we not set initial _and_ max to be safe?

Reply:
Yes, that would be better. I updated the PR accordingly.